chopper/lib/src/annotations.dart

```diff
@@ -433,7 +433,7 @@ final class Field {
 ///
 /// ```dart
 /// @Post(path: '/something')
-/// Future<Response> fetch(@FieldMap List<Map<String, dynamic>> query);
+/// Future<Response> fetch(@FieldMap Map<String, dynamic> query);
```

Comment:
Was this even supported?

Reply:
There is no support for `List<Map<String, dynamic>>` , I think it was a mistake.